WebContent/dashboard.jsp

```diff
@@ -0,0 +1,67 @@
+
+<%@page import="welldev.io.model.ProductList"%>
+<%@page import="java.util.ArrayList"%>
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+    pageEncoding="ISO-8859-1"%>
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="ISO-8859-1">
+<title>Dashboard</title>
+<meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+</head>
+<body>
+	<form action="addproduct" method="post" style="margin-top:30px; text-align: center;">
+		<input type="submit" value="Add Product">
+		<a href="<% if(session.getAttribute("un").toString().isEmpty()){
```

Comment:
i see the session check here.. good but you should check session in controller with filter.